google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -898,10 +898,6 @@ def default_acl
         # @see https://cloud.google.com/storage/docs/json_api/v1/buckets/setIamPolicy
         #   Buckets: setIamPolicy
         #
-        # @param [Boolean] force Force load the latest policy when `true`.
```

Comment:
@quartzmo, how do you feel about not introducing #reload! given the stale state shown in the example usage and instead provide an alias for `force:` being `reload:` as mentioned by remi?